Clicker/ViewControllers/GroupControlsViewController.swift

```diff
@@ -184,16 +146,46 @@ class GroupControlsViewController: UIViewController {
 extension GroupControlsViewController: ListAdapterDataSource {
 
     func objects(for listAdapter: ListAdapter) -> [ListDiffable] {
+        let numPolls = pollsDateAttendanceArray.reduce(0) { (result, pollsDateAttendanceModel) -> Int in
+            return result + pollsDateAttendanceModel.model.polls.count
+        }
+        
+        infoModel = GroupControlsInfoModel(numMembers: numMembers, numPolls: numPolls, code: session.code)
+        
+        attendanceHeader = HeaderModel(title: attendanceHeaderLabel)
+        pollSettingsHeader = HeaderModel(title: pollSettingsHeaderLabel)
+        
+        liveQuestionsSetting = PollsSettingModel(title: liveQuestionsTitle, description: liveQuestionsDescription, type: .liveQuestions, isEnabled: true)
+        filterSetting = PollsSettingModel(title: filterTitle, description: filterDescription, type: .filter, isEnabled: true)
+        locationSetting = PollsSettingModel(title: locationTitle, description: locationDescription, type: .filter, isEnabled: true)
+        
+        let spaceOne = SpaceModel(space: spaceOneHeight, backgroundColor: .clickerBlack1)
+        let spaceTwo = SpaceModel(space: spaceTwoHeight, backgroundColor: .clickerBlack1)
+        let spaceFour = SpaceModel(space: spaceThreeHeight, backgroundColor: .clickerBlack1)
+        
         if pollsDateAttendanceArray.count <= 3 {
-            return pollsDateAttendanceArray
+            let precursorArray: [ListDiffable] = [infoModel, spaceOne, attendanceHeader, spaceTwo]
+            let postArray: [ListDiffable] = [exportAttendanceModel, spaceFour, pollSettingsHeader, liveQuestionsSetting, filterSetting, locationSetting, spaceTwo]
+            let objectsArray = precursorArray + pollsDateAttendanceArray as [ListDiffable] + postArray
+            return objectsArray
```

Comment:
can we do `return [precursorArray, pollsDateAttendanceArray, postArray].flatMap {$0}`
https://developer.apple.com/documentation/swift/sequence/2905332-flatmap